shiny/www/py-shiny/brand/_brand-yml.scss

```diff
@@ -0,0 +1,205 @@
+
+  // If the brand foreground/background are close enough to black/white, we
+  // use those values. Otherwise, we'll mix the white/black from the brand
+  // fg/bg with actual white and black to get something much closer.
+  @return (
+    "white": if(contrast-ratio($white, white) <= 1.15, $white, mix($white, white, 20%)),
+    "black": if(contrast-ratio($black, black) <= 1.15, $black, mix($black, black, 20%))
+  );
+}
+
+/*-- scss:defaults --*/
+
+// Sass variables from `brand` will be inserted (above) here in this order:
+// * brand.color.palette
+// * brand.defaults
+// * brand.color
+// * brand.typography
+
+//*-- brand: initial defaults --*//
+$brand_color_foreground: null !default;
+$brand_color_background: null !default;
+$brand_color_primary: null !default;
+$brand_color_secondary: null !default;
+$brand_color_tertiary: null !default;
+$brand_color_success: null !default;
+$brand_color_info: null !default;
+$brand_color_warning: null !default;
+$brand_color_danger: null !default;
+$brand_color_light: null !default;
+$brand_color_dark: null !default;
+$brand_typography_base_family: null !default;
+$brand_typography_base_size: null !default;
+$brand_typography_base_line-height: null !default;
+$brand_typography_base_weight: null !default;
+$brand_typography_headings_family: null !default;
+$brand_typography_headings_line-height: null !default;
+$brand_typography_headings_weight: null !default;
+$brand_typography_headings_color: null !default;
+$brand_typography_headings_style: null !default;
+$brand_typography_monospace_family: null !default;
+$brand_typography_monospace_size: null !default;
+$brand_typography_monospace_weight: null !default;
+$brand_typography_monospace-inline_family: null !default;
+$brand_typography_monospace-inline_color: null !default;
+$brand_typography_monospace-inline_background-color: null !default;
+$brand_typography_monospace-inline_size: null !default;
+$brand_typography_monospace-inline_weight: null !default;
+$brand_typography_monospace-block_family: null !default;
+$brand_typography_monospace-block_line-height: null !default;
+$brand_typography_monospace-block_color: null !default;
+$brand_typography_monospace-block_background-color: null !default;
+$brand_typography_monospace-block_weight: null !default;
+$brand_typography_monospace-block_size: null !default;
+$brand_typography_link_background-color: null !default;
+$brand_typography_link_color: null !default;
+$brand_typography_link_weight: null !default;
+$brand_typography_link_decoration: null !default;
+
+//*-- brand.color --*//
+$body-color: $brand_color_foreground !default;
+$body-bg-dark: $brand_color_foreground !default;
+$body-bg: $brand_color_background !default;
+$body-color-dark: $brand_color_background !default;
+$body-secondary-color: $brand_color_secondary !default;
+$body-secondary: $brand_color_secondary !default;
+$body-tertiary-color: $brand_color_tertiary !default;
+$body-tertiary: $brand_color_tertiary !default;
```

Comment:
Maybe let's save this for another PR, but from a quick look, it seems `$body-secondary` and `$body-tertiary` don't exist?

Reply:
There's a good chance I copied this from the Quarto implementation. I'll check.

Reply:
Yup, came from here: https://github.com/quarto-dev/quarto-cli/blob/bf550cf7049f19d486b7410cb746b70367d0c359/src/core/sass/brand.ts#L31-L34

Happy to leave this for a follow-up PR.